StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoBodyView.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  GenericInfoBodyView.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 7/11/24.
+//
+
+import Foundation
+import UIKit
+
+func GenericInfoBodyView(
+    body: FinancialConnectionsGenericInfoScreen.Body?,
+    didSelectURL: @escaping (URL) -> Void
+) -> UIView? {
+    guard  let body, !body.entries.isEmpty else {
+        return nil
+    }
+    let verticalStackView = UIStackView()
+    verticalStackView.axis = .vertical
+    verticalStackView.spacing = 0
+    for entry in body.entries {
+        let entryView: UIView?
+        switch entry {
+        case .text(let textBodyEntry):
+            entryView = TextBodyEntryView(
+                textBodyEntry,
+                didSelectURL: didSelectURL
+            )
+        case .image(let image):
+            _ = image
+            entryView = nil // TODO(kgaidis): implement ImageBodyEntry support
+        case .unparasable:
+            entryView = nil // skip
+        }
+        if let entryView {
+            verticalStackView.addArrangedSubview(entryView)
+        }
+    }
+    // check `isEmpty` in case we were not able to handle any entry type
+    return verticalStackView.arrangedSubviews.isEmpty ? nil : verticalStackView
+}
+
+private func TextBodyEntryView(
+    _ textBodyEntry: FinancialConnectionsGenericInfoScreen.Body.TextBodyEntry,
+    didSelectURL: @escaping (URL) -> Void
+) -> UIView {
+    let font: FinancialConnectionsFont
+    let boldFont: FinancialConnectionsFont
+    let textColor: UIColor
+    switch textBodyEntry.size {
+    case .xsmall:
+        font = .body(.extraSmall)
+        boldFont = .body(.extraSmallEmphasized)
+        textColor = .textSubdued
+    case .small:
+        font = .body(.small)
+        boldFont = .body(.smallEmphasized)
+        textColor = .textSubdued
+    case .medium: fallthrough
+    case .unparsable: fallthrough
+    case .none:
+        font = .body(.medium)
+        boldFont = .body(.mediumEmphasized)
+        textColor = .textDefault
+    }
+    let textView = AttributedTextView(
+        font: font,
+        boldFont: boldFont,
+        linkFont: font,
+        textColor: textColor,
+        alignment: {
+            switch textBodyEntry.alignment {
+            case .center:
+                return .center
+            case .right:
+                return .right
+            case .left: fallthrough
```

Comment:
Should this be:
```suggestion
            case .left:
                return .left
```

Reply:
@mats-stripe could you expand on what you mean by this?

it should `fallthrough` to be what you suggested:

```
            case .left: fallthrough
            case .unparsable: fallthrough
            case .none:
                return .left
            }
```

should be same as (unless I'm missing something!):

```
            case .left:
                return .left
            case .unparsable:
                return .left
            case .none:
                return .left
            }
```

Reply:
confirmed on Slack that this should be OK 👍